### ISSUE TITLE
Get rid of single backtick for infix function application

### ISSUE THREAD

**Issue by pchiusano** [unisonweb/unison#2229]
Discussed on Slack a while back but never created a ticket to track.

The pretty-printer doesn't have any way of knowing when to use it when rendering, so it seems a bit pointless, more like a shortcut. 

Also a `append` b can be rewritten `a |> append <| b`
```
append = (Text.++)

> "Howdy" |> append <| " partner!"
          |> append <| " Yee haw!!"
          |> append <| " Unison's got infix wordy already!"
          |> append <| " Do we really need special syntax?"
```
Once this is done, we could maybe start using single backtick within a doc in place of the double backtick example syntax.

This should be pretty easy - open up `TermParser.hs` and just delete the parser for this. And then update all the tests that are using the old syntax.

**Comment by kolharsam** [comment]
Hey @pchiusano :wave:

Could I please get some help in fixing the broken tests? I am also not sure of the changes I've made on `TermParser.hs`.

**Comment by ChrisPenner** [comment]
Merged in #2570
